src/main/resources/com/rultor/agents/daemons/end.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+#@todo #754:30min Remove the deprecation code from the shell.
+# Move the code to Java. May be
```

Comment:
@mkordas May be the lack of my understanding of the project :)
- in the class `com.rultor.agents.daemons.EndsDaemon` and `com.rultor.agents.daemons.StartsDaemon` I need the image. I cannot find the flow.